### PR TITLE
Docs: examples and testing explanation

### LINKED ISSUE #50: Docs: write down how to use `test-builds` as examples
I explained how I'm developing this at https://github.com/readthedocs/readthedocs-client/issues/27#issuecomment-1495792410. It would be good to create a small page documenting this because it's going to be useful in the future.

### PR DESCRIPTION
**Pull request by humitos**
- Link to a bunch of live examples
- Explain how to test them locally with a development instance

Closes #50

### CODE REVIEW

**Comment by humitos** [comment]
I don't think this needs a review, but feel free to do it if you consider 😄

File: docs/conf.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-project = 'JavaScript client: "readthedocs-superpowers.js"'
+project = 'Read the Docs Addons"'
```
Comment (ericholscher) [inline docs/conf.py:1]:
```suggestion
project = 'Read the Docs Addons'
```

File: docs/index.rst [hunk 2]
```diff
@@ -8,7 +8,128 @@ This client allows projects to quickly integrate with all the nice features that
...
+
+Contents
+--------
+
+.. toctree::
+
+   releasing
+   testing
+
+.. raw:: html
```
Comment (ericholscher) [inline docs/index.rst:69]: Should these be injected JS files? I guess it's fine to hack it for now :)

**Review by ericholscher — APPROVED** [review]

File: docs/index.rst (hunk 2 above)
Comment (humitos) [reply docs/index.rst:69]: Yeah, it was quick and dirty... but eventually this should be nicer 😄

**Pull request merged**